core/logger/logger.go

```diff
@@ -52,6 +52,7 @@ func init() {
 var _ relaylogger.Logger = (Logger)(nil)
 
 //go:generate mockery --quiet --name Logger --output . --filename logger_mock_test.go --inpackage --case=underscore
+//go:generate mockery --quiet --name Logger --output ./mocks/ --case=underscore
```

Comment:
is this mock different from the one above? or are only the names different?

Reply:
The first go:generate command seems to output a test file (*_test.go), which I'm not able import in other modules.

Thats why I had to generate a mock struct for the interface into a separate mocks package, just like we do for BHS and Timer